.github/workflows/ci.yml

```diff
@@ -79,7 +80,19 @@ jobs:
                 if: matrix.calculate-code-coverage == 'yes'
                 env:
                     COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                    COVERALLS_PARALLEL: true
+                    COVERALLS_FLAG_NAME: "PHP ${{ matrix.php }} + Symfony@${{ matrix.symfony || '*' }}"
                 run: bin/php-coveralls -v --exclude-no-stmt
+
+    coveralls-finish:
+        needs: tests
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Notify Coveralls when build is finished
```

Comment:
I think we should bring a command to our SDK instead (maybe as new PR?)

Reply:
Yes, I was surprised when not found possibility to finish parallel builds out of box.
But it should be new PR anyway

Reply:
maybe I can ask you to propose a PR for this new functionality? :)